modules/tinymce/CHANGELOG.md

```diff
@@ -24,6 +24,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - The `onSetup` api function would not run when defining custom group toolbar button. #TINY-9496
 - An element could be dropped onto the decendants of a noneditable element. #TINY-9364
 - Checkmark did not show in menu colorswatches. #TINY-9395
+- The foreground and background menu icons would not properly update to display the last used color. #TINY-9497
```

Comment:
You also introduced a new `setIconFill` api to all nested menu items so a separate line for that would be nice.